MAUI/DataForm/ManualDataFormItem.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: post
+title: Manually create DataFormItems in Maui DataForm control | Syncfusion
+description: Learn here all about Working with manually created DataFormItem's in Syncfusion .NET MAUI DataForm(SfDataForm) control.
```

Comment:
Learn here all about Working with manually created DataFormViewItems in Syncfusion .NET MAUI DataForm(SfDataForm) control.